src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py

```diff
@@ -53,7 +53,13 @@ def get(self, request: Request, organization) -> Response:
         stats_period = min(
             parse_stats_period(request.GET.get("statsPeriod", "24h")), timedelta(days=2)
         )
+
         end_time = timezone.now()
+        # Quantize time boundary down so that during a 5-minute interval, the query time boundaries
+        # remain the same to leverage the snuba cache
```

Comment:
nit: few small examples would be good to add in comments:

Reply:
It is a one liner @andriisoldatenko , I am sure anyone will understand what is going on just from looking at this line